external-crates/move/documentation/book/src/abilities.md

```diff
@@ -7,6 +7,11 @@ the notion of storage for the blockchain). This is implemented by gating access
 instructions so that for a value to be used with the bytecode instruction, it must have the ability
 required (if one is required at all—not every instruction is gated by an ability).
 
+For Sui, `key` is used to signify an [object](./abilities/object.md). Objects are the basic unit of
+storage where each object has a unique, 32-byte ID. `store` is then used to both indicate what data
+can be stored inside of an object, and is also used to indicate what types can be transferred
+outside of their defining module.
+
```

Comment:
To call attention to these entries, what if we block-quoted them?

I think another idea is something like the warning span in `mdbook`, but not as a warning -- as a sui-specific note.

Reply:
@damirka do you know if we have the ability to do custom quote blocks or anything in mdbook?

Otherwise, I'd say its a good suggestion, but maybe overkill at the moment? Since Sui is the only applicable chain for these docs at the moment